### PR TITLE
feat(clerk-js): Align the UserProfile with the designs

### PR DESCRIPTION
**Pull request by anagstef**
## Description

Before:
Screenshot 2024-01-15 at 23 42 50

After:

Screenshot 2024-01-16 at 01 33 03

## Checklist

- [ ] `npm test` runs as expected.
- [ ] `npm run build` runs as expected.
- [ ] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [ ] (If applicable) [Documentation](https://github.com/clerk/clerk-docs) has been updated

## Type of change

- [ ] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

## Packages affected

- [ ] `@clerk/backend`
- [ ] `@clerk/chrome-extension`
- [ ] `@clerk/clerk-js`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/fastify`
- [ ] `gatsby-plugin-clerk`
- [ ] `@clerk/localizations`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/remix`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/shared`
- [ ] `@clerk/themes`
- [ ] `@clerk/types`
- [ ] `build/tooling/chore`

### CODE REVIEW

**Comment by anagstef** [comment]
!preview

**Comment by clerk-cookie** [comment]
Hey @anagstef, your preview is currently deploying and should be available shortly.

| Status | Preview | Updated (UTC) |
| :-- | :-- | :-- |
| :cook: _Deploying..._ | [Inspect](https://javascript-preview-br1ybbz8f.clerkstage.dev) | Jan 16, 2024 01:30 PM |

**Review by octoper — APPROVED** [review]
💯

File: packages/clerk-js/src/ui/foundations/sizes.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ const dynamicSpaceUnits = Object.freeze({
   '2': '0.5rem',
   '2x5': '0.625rem',
   '3': '0.75rem',
-  '2x25': '0.8125rem',
```
[the remark this answers is not in the document]
Comment (anagstef) [reply packages/clerk-js/src/ui/foundations/sizes.ts:14]: Yes, not used anymore and it feels weird! 😆

**Pull request merged**